pkg/util/steps/runner.go

```diff
@@ -21,7 +21,7 @@ import (
 
 // FriendlyName returns a "friendly" stringified name of the given func.
```

Comment:
This is not a part of your change, but it's good time to have more specific comment here.

Reply:
done